RadarSDK/Radar.h

```diff
@@ -431,17 +439,47 @@ typedef void(^ _Nonnull RadarRouteCompletionHandler)(RadarStatus status, RadarRo
 
  @param near The location to search.
  @param radius The radius to search, in meters. A number between 100 and 10000.
- @param tags An array of tags to filter. See https://radar.io/documentation/geofences
```

Comment:
why was this removed? seems like the link would be helpful?

Reply:
no idea why, will revert.